### DISCUSSION TITLE
IOS App Amp 4.4.3 vs. 5.x

### DISCUSSION THREAD

**Discussion by vontani** [ampache/ampache#3103] [General]
dear,

i did just install Amp 5.1 and had no chance to connect at all to 5.1, i did try to play around with configs here and therebut no change, isub, substreamer  etc. was used to get my music with me.. how ever, i waist hours to and ended back to 4.4.3 and this works out of the box.

changes made to 5.1 that makes me cry ! what's the solution ?

**Comment by lachlan-00** [comment]
sounds like ampache 5.2 will help you as the api changed between 5 and 4 releases.

5.2 will support all previous ampache api clients (api3, api4 and the new api5)

**Comment by vontani** [comment]
what happen to 5.1 ? as i understand, 4.4.3 is an older version, 5.1 the newest release, right ? i would like to understand where the problem is and why ampache starts be difficult as soon it comes to ios apps..
